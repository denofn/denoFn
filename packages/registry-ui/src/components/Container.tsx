import React from "react";
import { css } from "otion";

export function Container({ children }: React.PropsWithChildren<{}>) {
  return (
    <div
      className={css({
        display: "flex",
        padding: "1.5rem",
      })}
    >
      {children}
    </div>
  );
}
