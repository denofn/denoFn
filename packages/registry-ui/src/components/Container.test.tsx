import { render, cleanup } from "@testing-library/react";
import React from "react";

import { Container } from "./Container";

afterEach(cleanup);

it("renders with the correct styles", () => {
  const { asFragment } = render(<Container />);
  expect(asFragment()).toMatchSnapshot();
});
