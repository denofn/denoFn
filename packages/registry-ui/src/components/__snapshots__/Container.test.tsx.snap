// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with the correct styles 1`] = `
<DocumentFragment>
  <div
    class="_1m3j4vw"
  />
</DocumentFragment>
`;
